models/helpers.ts

```diff
@@ -244,6 +272,20 @@ function getSubmittableDocStatus(doc: RenderData | Doc) {
     return getInvoiceStatus(doc);
   }
 
+  if (
+    [ModelNameEnum.Shipment, ModelNameEnum.PurchaseReceipt].includes(
+      doc.schema.name as ModelNameEnum
```

Comment:
should've used `doc.schemaName`

Reply:
`doc.schemaName` is not accessible here.

I referred to the code above,
https://github.com/frappe/books/blob/544b8a14b1d93a96ce9ad7d856aeff2645427e94/models/helpers.ts#L239-L243